test/v2/generators/DepartureMessageGeneratorSpec.scala

```diff
@@ -68,6 +70,18 @@ class DepartureMessageGeneratorSpec extends AnyFreeSpec with Matchers with Optio
         validate("cc029c", generator.generate(departureId)(ReleaseForTransit))
       }
     }
+
+    "when supplied with message type " - {
```

Comment:
Please add "when supplied with message type RejectionFromOfficeOfDeparture with this message"